src/main/java/cc/polyfrost/oneconfig/internal/command/OneConfigCommand.java

```diff
@@ -32,104 +32,85 @@
-                    else builder.append(ChatColor.RED);
-                    builder.append(profile);
-                }
-                UChat.chat(builder.toString());
+    @Command(value = "Profile", description = "Actions related to profiles.", aliases = {"profiles"})
+    private static class Profile {
+        @Descriptor(value = "list", description = "View all profiles", aliases = {"view"})
+        private void list() {
+            StringBuilder builder = new StringBuilder()
+                    .append(ChatColor.GOLD).append("Available profiles:");
+            for (String profile : Profiles.getProfiles()) {
+                builder.append("\n");
+                if (OneConfigConfig.currentProfile.equals(profile)) builder.append(ChatColor.GREEN);
+                else builder.append(ChatColor.RED);
+                builder.append(profile);
             }
+            UChat.chat(builder.toString());
         }
 
-        @SubCommand(value = "switch", description = "Switch to a profile", aliases = {"enable", "set", "load"})
-        private static class SwitchProfile {
-            @Main
-            private static void main(@Name("profile") @Greedy String profile) {
-                if (!Profiles.doesProfileExist(profile)) {
-                    UChat.chat(ChatColor.RED + "The profile \"" + profile + "\" does not exist!");
-                } else {
-                    Profiles.loadProfile(profile);
-                    UChat.chat(ChatColor.GREEN + "Switched to the \"" + profile + "\" profile.");
-                }
+        @Descriptor(value = "switch", description = "Switch to a Profile", aliases = {"enable", "set", "load", "switch"})
+        private void switchProfile(@Descriptor("Profile Name") @Greedy String profile) {
+            if (!Profiles.doesProfileExist(profile)) {
+                UChat.chat(ChatColor.RED + "The Profile \"" + profile + "\" does not exist!");
+            } else {
+                Profiles.loadProfile(profile);
+                UChat.chat(ChatColor.GREEN + "Switched to the \"" + profile + "\" Profile.");
             }
         }
 
-        @SubCommand(value = "create", description = "Create a new profile", aliases = {"make"})
-        private static class Create {
-            @Main
-            private static void main(@Name("profile") @Greedy String profile) {
-                if (Profiles.doesProfileExist(profile)) {
-                    UChat.chat(ChatColor.RED + "The profile \"" + profile + "\" already exists!");
-                } else {
-                    Profiles.createProfile(profile);
-                    if (Profiles.doesProfileExist(profile)) Profiles.loadProfile(profile);
-                    UChat.chat(ChatColor.GREEN + "Created the \"" + profile + "\" profile.");
-                }
+        @Descriptor(value = "", description = "Create a new Profile", aliases = {"make"})
+        private void create(@Descriptor("Profile Name") @Greedy String profile) {
```

Comment:
@ Descriptor could be replaced by @ Description (here and in other places like it)